tern/utils/container.py

```diff
@@ -63,25 +63,25 @@ def check_container():
 
 def check_image(image_tag_string):
     '''Check if image exists'''
-    logger.debug("Checking if image {} is available on disk...".format(
-        image_tag_string))
+    logger.debug("Checking if image \"%s\" is available on disk...",
+        image_tag_string)
```

Comment:
```
>>> logger.debug("can I do %s ?", "this")
can I do this ?
```
Why are the escapes needed?

Reply:
It's generally good practice to include non-numeric values in quotes when logging, etc.  It makes it a little easier when scanning logs to see what value is key.

Reply:
Oh I see. Here the log message looks like this: `Checking if image "blah" is available on disk...`

Reply:
Yep.